fvtest/lljbtest/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+###############################################################################
+# Copyright (c) 2020, 2020 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at http://eclipse.org/legal/epl-2.0
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+###############################################################################
+
+find_package(LLVM 8 REQUIRED CONFIG)
+
+# we need clang to generate IR from C++
+find_program(CLANG_EXECUTABLE  NAMES clang)
+
+if (NOT CLANG_EXECUTABLE)
+    message(SEND_ERROR "Could not find clang")
```

Comment:
If it were me, I'd have used `FATAL_ERROR` instead. PS, you might want to factor this code out into a `FindClang.cmake` module. And, you might want to use the current compiler as a hint.

Reply:
I have changed this to `FATAL_ERROR` as suggested. As for factoring this out to a separate module, I would like to try that in a follow up PR instead of this one.